Doc/whatsnew/3.13.rst

```diff
@@ -143,6 +143,16 @@ opcode
   documented or exposed through ``dis``, and were not intended to be
   used externally.
 
+os
+--
+
+* Add :func:`os.timerfd_create`, :func:`os.timerfd_settime`, :func:`os.timerfd_gettime`, :func:`os.timerfd_settime_ns`,
+  and :func:`os.timerfd_gettime_ns` to provide a low level interface for Linux's timer notification file descriptor.
+  (Contributed by Masaru Tsuchiyama in :gh:`108277`.)
+
+* Add :const:`os.TFD_NONBLOCK`, :const:`os.TFD_CLOEXEC`, :const:`os.TFD_TIMER_ABSTIME`, and :const:`os.TFD_TIMER_CANCEL_ON_SET`
+  (Contributed by Masaru Tsuchiyama in :gh:`108277`.)
```

Comment:
Are these distinct from the timerfd interface?

Reply:
@AA-Turner 
This is a part of the timerfd interface.
Should I remove?

Reply:
@AA-Turner 
Merged the sections.